packages/fxa-settings/src/components/HeaderLockup/index.tsx

```diff
@@ -29,9 +30,13 @@ export const HeaderLockup = () => {
       }
     }
 
-    window.addEventListener('scroll', handleScroll);
+    const throttledHandler = throttle(() => {
+      handleScroll();
+    }, 100);
```

Comment:
> Note, because of this you now can easily scroll to the top and not have the shadow disappear, which is a little disappointing

Heh, we actually need a debounce to fix this. I've worked on an app where we had to throttle _and_ debounce, might need something similar here.